packages/db/tests/models/test-granules-executions-model.js

```diff
@@ -0,0 +1,194 @@
+const test = require('ava');
+const cryptoRandomString = require('crypto-random-string');
+
+const {
+  CollectionPgModel,
+  destroyLocalTestDb,
+  ExecutionPgModel,
+  fakeCollectionRecordFactory,
+  fakeExecutionRecordFactory,
+  fakeGranuleRecordFactory,
+  generateLocalTestDb,
+  GranulePgModel,
+  GranulesExecutionsPgModel,
+  tableNames,
+} = require('../../dist');
+
+const { migrationDir } = require('../../../../lambdas/db-migration');
+
+const testDbName = `granule_${cryptoRandomString({ length: 10 })}`;
+
+test.before(async (t) => {
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+
+  t.context.granulePgModel = new GranulePgModel();
+  t.context.granulesExecutionsPgModel = new GranulesExecutionsPgModel();
+
+  const collectionPgModel = new CollectionPgModel();
+  [t.context.collectionCumulusId] = await collectionPgModel.create(
+    t.context.knex,
+    fakeCollectionRecordFactory()
+  );
+
+  t.context.executionPgModel = new ExecutionPgModel();
+});
+
+test.beforeEach(async (t) => {
+  const [executionCumulusId] = await t.context.executionPgModel.create(
+    t.context.knex,
+    fakeExecutionRecordFactory()
+  );
+  t.context.executionCumulusId = executionCumulusId;
+  const [granuleCumulusId] = await t.context.granulePgModel.create(
+    t.context.knex,
+    fakeGranuleRecordFactory({
+      collection_cumulus_id: t.context.collectionCumulusId,
+    })
+  );
+  t.context.granuleCumulusId = granuleCumulusId;
+  t.context.joinRecord = {
+    execution_cumulus_id: executionCumulusId,
+    granule_cumulus_id: Number(granuleCumulusId),
+  };
+});
+
+test.afterEach.always(async (t) => {
+  await t.context.knex(tableNames.granulesExecutions).where(t.context.joinRecord).del();
```

Comment:
Is this strictly required, given each run does a database create and the factory should randomize the ID, and the teardown wipes the entire database?       Do we need to make the crypto length longer given concerns mentioned below?

Reply:
Turns out that it is not necessary. Removing

Reply:
b7c15e5008ac84debf877650ffcd75128065fb00